.github/workflows/evaluate.yml

```diff
@@ -0,0 +1,86 @@
+name: Run evaluation
+
+on:
+  push:
+    branches: [ master, docker ]
```

Comment:
I would like to run testing on all branches and evaluation (snakemake) only on tagged commits on master

Reply:
Should be solved -- see `run_benchmark.yml`. For now, I still configured it to also run the benchmark if commits are made to a branch called `test_benchmark`, so that you can try out the script before making a tagged commit. Change it however you like :)